e2e/test/config/TestConfig.cs

```diff
@@ -5,7 +5,7 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
 //Workers = 0 makes the test engine use one worker per available core. It does not mean to run serially.
-[assembly: Parallelize(Workers = 0, Scope = ExecutionScope.ClassLevel)]
+//[assembly: Parallelize(Workers = 0, Scope = ExecutionScope.ClassLevel)]
```

Comment:
Delete?

Reply:
yes, this class will be deleted once E2EMsBaseTest can handle logging as well. I'll add that change once a run is complete on VSTS